Portal/src/Datahub.Infrastructure/Services/RequestManagementService.cs

```diff
@@ -125,6 +126,13 @@ await ctx.Entry(project)
             var workspaceDefinition =
                 await _resourceMessagingService.GetWorkspaceDefinition(project.Project_Acronym_CD,
                     requestingUser.Email);
+
+            if (isDelete) {
+                foreach (var template in allTemplates) {
+                    if (template.Name == terraformTemplate) template.Status = "delete";
+                }
+            }
+            
```

Comment:
The status here won't be persisted to the database. It would make more sense to process this up near L108 so that the `ProcessRequest` call can persist the template status into the database

Reply:
now persisted